aten/src/ATen/core/jit_type.h

```diff
@@ -542,6 +542,8 @@ struct CAFFE2_API TensorType : public Type {
       at::Device device,
       at::IntArrayRef sizes);
 
+  static TensorTypePtr createInferred();
```

Comment:
This is only used in one place (`getInferred`), so I think we should not have a separate function for it for now. Let's inline it.